helper.go

```diff
@@ -54,6 +57,22 @@ func MGetCache(client Client, ctx context.Context, ttl time.Duration, keys []str
 	return doMultiCache(client, ctx, cmds.s, keys)
 }
 
+func isCacheDisabled(client Client) bool {
+	switch c := client.(type) {
+	case *singleClient:
+		// what do we do here ? @rueian
```

Comment:
Is this the right approach to this? @rueian 
Can you suggest?

Reply:
We may copy the `DisableCache` to a new bool field in the `singleClient` and then check the field here.